docs/src/modules/ROOT/pages/using-timefold-solver/modeling-planning-problems.adoc

```diff
@@ -2839,3 +2839,240 @@ def generate_demo_data(demo_data: DemoData) -> Timetable:
 
 Usually, most of this data comes from your data layer,
 and your solution implementation just aggregates that data and creates the uninitialized planning entity instances for the solver to plan.
+
+[#planningModelInheritance]
+== Inheritance
```

Comment:
We'll soon be doing the same exercise for solution. :-) I'll be explicit here and say that this is inheritance of entities and variables.